client/landing/jetpack-cloud/sections/backups/index.js

```diff
@@ -29,6 +31,7 @@ import {
 
 export default function () {
 	if ( config.isEnabled( 'jetpack-cloud/backups' ) ) {
+		FullStory.init( { orgId: TRACKING_IDS.jetpackCloudFullStory } );
```

Comment:
Is the intention to use FullStory for all Jetpack Cloud users, across all environments, including dev and staging? If so, this is fine, but otherwise you're going to need to use a flag.